### PR TITLE
Configured CORS header "Access-Control-Expose-Headers" to APIM

### PR DESCRIPTION
**Pull request by HiranyaKavishani**
## Purpose
Enable CORS header "Access-Control-Expose-Headers" configuration in to APIM

Fixed #4461

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/CORSRequestHandler.java [hunk 1]
```diff
@@ -105,6 +106,9 @@ void initializeHeaders() {
                 allowedMethodList = Arrays.asList(allowedMethods.split(","));
             }
         }
+        if(exposeHeaders == null) {
```
Comment (bhathiya) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/CORSRequestHandler.java]: Please format

**Comment by bhathiya** [comment]
@HiranyaKavishani please check why Travis has failed.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java [hunk 2]
```diff
@@ -6583,6 +6583,16 @@ public static String getAllowedMethods() {
...
     }
 
+    /**
+     * Used to get access control expose headers define in api-manager.xml
+     *
+     * @return access control expose headers string
+     */
+    public static String getExposedHeaders() {
+        return ServiceReferenceHolder.getInstance().getAPIManagerConfigurationService().getAPIManagerConfiguration().
+                getFirstProperty(APIConstants.CORS_CONFIGURATION_ACCESS_CTL_EXPOSE_HEADERS);
```
Comment (sanjeewa-malalgoda) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java]: getAccessControlExposedHeaders?

Comment (HiranyaKavishani) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java]: Fixed with a841f9c011fcd78b4a9d50a31f600557c5c9420c

**Pull request merged**